src/server/config/routes.js

```diff
@@ -1,75 +1,30 @@
-const axios = require('axios');
 const Router = require('koa-router');
-const jwt = require('jsonwebtoken');
-const _ = require('lodash');
 
-const config = require('config');
-
-const { logger } = global;
+const {
+  isDev, landingLoginUrl, webSocketUrl, apiUrl,
```

Comment:
Please don't use destructuring on config importing, import full config and access through it